bitnami/harbor/templates/notary/notary-secret.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.notary.enabled }}
```

Comment:
The certificate are mounted as volumes so I don't see the need to put the path into a secret.

Reply:
We moved server and signer jsons here because they stored our external DB password in plain within storage.db_url, same as happened with clair.

About the certificates, having tls.key in a Secret fits much more than in a Configmap.

Reply:
Not sure about this point, shouldn't it be the same case as the password above?